### DISCUSSION TITLE
lifestyle/will-time-heal-everything/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#54] [Announcements]
# lifestyle/will-time-heal-everything/

随着时间的流逝，你每天都在为自己的未来做积极的改变，虽然这个进度有慢有快，但是你是一个向上走的状态。为了让自己的快乐重新回来，你在努力，这样的时间会把你治愈的

https://zxuqian.cn/lifestyle/will-time-heal-everything/

**Comment by szeminglee** [comment]
无中生友

**Comment by DoooReyn** [comment]
朋友，时间不会治愈一切，只能减弱关联，等到想起的那一天，你才会知道扯掉这块硬邦邦的痂，下面是条你曾经不敢面对的疤。

**Comment by kuizuo** [comment]
时间不一定能治愈一切，但不断学习进步可以。
